### ISSUE TITLE
"TeX capacity exceeded" when using allowframebreaks

### ISSUE THREAD

**Issue by gmbrauer** [matze/mtheme#20]
This MWE:

````
\documentclass[titleprogressbar]{beamer}
\usetheme{m}
\usepackage{lipsum}
\begin{document}

\begin{frame}[allowframebreaks]
\frametitle{A Frame Title}
\lipsum
\end{frame}

\end{document}
````

bombs with 

````
...
Runaway argument?
! TeX capacity exceeded, sorry [main memory size=5000000].
\reserved@a #1#2->\let #2#1\reserved@a 
                                       
l.9 \end{frame}
...
````

**Comment by matze** [comment]
Good catch. I don't know what to do about it right now but I will investigate.

**Comment by boppreh** [comment]
I just hit this problem too.

**Comment by big-r81** [comment]
I just hit this problem too.
I need [allowframbreaks] for the \printbibliography command

**Comment by rchurchley** [comment]
This issue has now been fixed as of #58 and can probably be closed (independently of #59)
